dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -1517,6 +1525,873 @@ public void findByObjectAndFeatureInternalServerErrorTest() throws Exception {
+            .param("feature", trueForAdmins.getName()))
+            .andExpect(jsonPath("$.page.totalElements", is(0)));
+
+        // verify that it works for administators inspecting other users - by using the eperson parameter
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.item")
+            .param("uuid", itemId)
+            .param("uuid", secondItemId)
+            .param("feature", alwaysFalse.getName())
+            .param("feature", trueForAdmins.getName())
+            .param("eperson", eperson.getID().toString()))
+            .andExpect(jsonPath("$.page.totalElements", is(0)));
+
+        // verify that it works for administators inspecting other users - by assuming login
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.item")
+            .param("uuid", itemId)
+            .param("uuid", secondItemId)
+            .param("feature", alwaysFalse.getName())
+            .param("feature", trueForAdmins.getName())
+            .header("X-On-Behalf-Of", eperson.getID()))
+            .andExpect(jsonPath("$.page.totalElements", is(0)));
+
+        // verify that it works for anonymous users
+        getClient().perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.item")
+            .param("uuid", itemId)
+            .param("uuid", secondItemId)
+            .param("feature", alwaysFalse.getName())
+            .param("feature", trueForLoggedUsers.getName()))
+            .andExpect(jsonPath("$.page.totalElements", is(0)));
+    }
+
+    @Test
+    /**
+     * Verify that the find by multiple objects and features
+     * return the 204 No Content code when the requested object doesn't exist but the uri
+     * is potentially valid (i.e. deleted object) or the feature doesn't exist
+     *
+     * @throws Exception
+     */
+    public void findByNotExistingMultipleObjectsAndFeatureTest() throws Exception {
+        UUID wrongSiteId = UUID.randomUUID();
+        String siteId = wrongSiteId.toString();
+        String wrongSiteUri = "http://localhost/api/core/sites/" + wrongSiteId;
+        Site site = siteService.findSite(context);
+        SiteRest siteRest = siteConverter.convert(site, DefaultProjection.DEFAULT);
+        String siteUri = utils.linkToSingleResource(siteRest, "self").getHref();
+
+        // disarm the alwaysThrowExceptionFeature
+        configurationService.setProperty("org.dspace.app.rest.authorization.AlwaysThrowExceptionFeature.turnoff", true);
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        // verify that it works for administrators, no result - with eperson parameter
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.site")
+            .param("uuid", siteId)
+            .param("uuid", siteId)
```

Comment:
You duplicated this `uuid` param here (and in the next 13 tests). It'd also help make this test clearer if you rename `siteId` to `wrongSiteId` or similar, to make it clear it's invalid